Add release note: Pantheon's next-generation Global CDN is now generally available (#10154)
pantheon-systems/documentation: src/source/releasenotes/2026-07-13-global-cdn-ga.md

---
title: "Pantheon's next-generation Global CDN is now generally available"
published_date: "2026-07-13"

> published_at: "2026-07-13T14:00:00Z"

Reviewer: ```suggestion
published_at: "2026-07-13T21:34:07Z"
```

> published_at: "2026-07-13T21:34:07Z"

Reviewer: ```suggestion
published_date: "2026-07-15"
published_at: "2026-07-15T14:54:28Z"
```

- **Platform vanity domain support** — Organization-owned [vanity hostnames](/guides/domains/vanity-domains) (e.g., `live-mysite.example-agency.com`) are fully supported. Migration for these sites is owned by our Professional Services team and coordinated through support — [open a support ticket](/guides/support/contact-support/) to get started.

## Action Required

Eligible sites will display a GCDN banner in the Pantheon dashboard. To migrate:

1. Locate the GCDN banner on an eligible site.
2. Click it and complete the guided process.
3. Point your DNS records to the new GCDN infrastructure.

Domain verification supports DNS-01 TXT record validation, requiring TXT records added at your DNS provider.

## Eligibility

GCDN is available to all sites on the platform except those currently using [Advanced Global CDN (AGCDN)](/guides/agcdn). AGCDN customers will be migrated in a future phase — no action is required from them at this time.

## More information

> For full details on GCDN, refer to the [Global CDN guide](/guides/global-cdn/global-cdn-beta) on the Pantheon docs site.

Reviewer: ```suggestion
For full details on GCDN, refer to the [Global CDN guide](/guides/global-cdn/next-gen-global-cdn) on the Pantheon docs site.
```
